plugins/draw-tools.js

```diff
@@ -1,8 +1,8 @@
 // @author         breunigs
 // @name           Draw tools
 // @category       Draw
-// @version        0.8.1
-// @description    Allow drawing things onto the current map so you may plan your next move.
+// @version        0.8.0
```

Comment:
Perhaps here again, last commit was not careful enough

Reply:
right and not.
the provided PRs are trying to be independant to keep them "one per feature".
As mentioned several times in the paste these two lines need to be updated manually depending on the selected order of merging.